### PR TITLE
vrftr-3340 vrftr-5848 Add delete examples

### PR DESCRIPTION
**Pull request by smediavi**

### CODE REVIEW

File: net-core-with-file-api-sdk/FileAPI.MFT.FileSystem.NetCore22/Examples/Delete.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            #region Custom parameters
+
+            var tenantId = "MyTenantId"; // Only necessary for multi-tenant token.
+
+            #endregion
+
+            _output.WriteTittle("Executing Streaming.SDK example: Set subscriber file to 'deleted'");
+
+            // Configure the file id.
+            var fileId = "FileId";
```
Comment (VismaAlbertoInfante) [inline net-core-with-file-api-sdk/FileAPI.MFT.FileSystem.NetCore22/Examples/Delete.cs]: Place the fileId inside the `Custom parameters` section.

File: net-core-with-file-api-sdk/FileAPI.MFT.FileSystem.NetCore22/Examples/Delete.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+            // Delete the file.
+            var deleteResult = await FileSystem.DeleteFileAsync(fileId: fileId, tenantId: tenantId);
+
+            Assert.True(deleteResult);
+
+            // Print the result.
+            _output.WriteLine($"File could be deleted:{deleteResult}");
```
Comment (VismaAlbertoInfante) [inline net-core-with-file-api-sdk/FileAPI.MFT.FileSystem.NetCore22/Examples/Delete.cs]: Add a whitespace after the `:`.

File: net-core-with-file-api-sdk/FileAPI.MFT.FileSystem.NetCore22/Examples/Delete.cs (hunk 1 above)
Comment (smediavi) [reply net-core-with-file-api-sdk/FileAPI.MFT.FileSystem.NetCore22/Examples/Delete.cs]: Done.

File: net-core-with-file-api-sdk/FileAPI.MFT.FileSystem.NetCore22/Examples/Delete.cs (hunk 2 above)
Comment (smediavi) [reply net-core-with-file-api-sdk/FileAPI.MFT.FileSystem.NetCore22/Examples/Delete.cs]: Done.

**Review by VismaAlbertoInfante — APPROVED** [review]

**Pull request merged**